src/index.html

```diff
@@ -5,8 +5,313 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="header__nav">
```

Comment:
nb: you can add one more `Block` class to navigation like `class="header__nav nav"` and use `nav__list`, `nav__item` etc